homeassistant/components/tibber/__init__.py

```diff
@@ -46,6 +48,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     )
     hass.data[DOMAIN] = tibber_connection
 
+    async_setup_services(hass)
```

Comment:
Please move the setup of services to `async_setup`. This ensures services are always there, even if an entry isn't loading or fails to load.